### PR TITLE
Include URL and UUID in Home Assistant controller messages

### PR DESCRIPTION
**Pull request by emontnemery**
Can't be merged until https://github.com/home-assistant/frontend/pull/12354 is merged and an updated app is published

### CODE REVIEW

File: pychromecast/controllers/homeassistant.py [hunk 1]
```diff
@@ -45,6 +47,7 @@ def hass_connected(self):
             self.status is not None
             and self.status["connected"]
             and self.status["hassUrl"] == self.hass_url
+            and self.status["hassUUID"] == self.hass_uuid
```
Comment (balloob) [inline pychromecast/controllers/homeassistant.py:50]: Why is this necessary? In what case are the URLs the same but the UUIDs not?

Comment (emontnemery) [reply pychromecast/controllers/homeassistant.py:50]: When developing or testing with multiple hass instances this can be the case.

**Review by balloob — APPROVED** [review]
